cg/services/order_validation_service/rules/case_sample/rules.py

```diff
@@ -406,3 +408,18 @@ def validate_case_names_different_from_sample_names(
                 )
                 errors.append(error)
     return errors
+
+
+def validate_not_all_samples_unknown_in_case(
+    order: OrderWithCases, **kwargs
+) -> list[AllSamplesUnknownStatusError]:
+    errors: list[AllSamplesUnknownStatusError] = []
+
+    for case_index, case in order.enumerated_cases:
```

Comment:
```suggestion
    for case_index, case in order.enumerated_new_cases:
```

Reply:
We do not need to consider old cases since the status will not be updated for their samples